packaging/ubuntu-16.04/snapd.maintscript

```diff
@@ -3,3 +3,4 @@
 rm_conffile /etc/grub.d/09_snappy 1.7.3ubuntu1
 rm_conffile /etc/ld.so.conf.d/snappy.conf 2.0.7~
 rm_conffile /etc/apparmor.d/usr.lib.snapd.snap-confine 2.23.6~ snap-confine
+rm_conffile /etc/apparmor.d/usr.lib.snapd.snap-confine
```

Comment:
The line before this is the one that was not obviously correct in the previous SRU, it's even less obvious to me why we would have two different rm_conffile lines here.  As far as I know, the second line is a superset of the first.